crates/sui-package-resolver/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "sui-package-resolver"
+version = "0.1.0"
+edition = "2021"
+authors = ["Mysten Labs <build@mystenlabs.com>"]
+license = "Apache-2.0"
+publish = false
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+async-trait.workspace = true
+bcs.workspace = true
+lru.workspace = true
+diesel.workspace = true
+move-binary-format.workspace = true
+move-core-types.workspace = true
+rocksdb.workspace = true
```

Comment:
I'd like to avoid taking this dependency for the GraphQL service.  Can we split out the implementations of the various stores so that the analytics pipeline implements the local db store and has the `rocksdb` dependency, and GraphQL implements the existing `DbPackageStore` based on postgres/diesel?